services/director-v2/src/simcore_service_director_v2/core/settings.py

```diff
@@ -352,7 +364,7 @@ class AppSettings(BaseCustomSettings, MixinLoggingSettings):
 
     TRAEFIK_SIMCORE_ZONE: str = Field("internal_simcore_stack")
 
-    DASK_SCHEDULER: DaskSchedulerSettings = Field(auto_default_from_env=True)
+    DASK_SCHEDULER: DaskComputationalBackendSettings = Field(auto_default_from_env=True)
```

Comment:
ame here, this should be ``DIRECTOR_V2_DASK_SCHEDULER``

Reply:
renamed